man/systemd.xml

```diff
@@ -822,6 +829,13 @@
         <option>--log-target=</option>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>$SYSTEMD_LOG_TIME</varname></term>
+        <listitem><para>Controls whether systemd prefixes log
+        messages with the current time. This can be overridden with
+        <option>--log-time=</option>.</para></listitem>
```

Comment:
No "=". (We generally skip it, if the argument is not mandatory.)

Reply:
ack